### PR TITLE
Update PHP 5.6 doc for Tuleap 9.14

### PR DESCRIPTION
**Pull request by vaceletm**

### CODE REVIEW

File: languages/en/administration-guide/run-php56.rst [hunk 1]
```diff
@@ -3,62 +3,79 @@
...
+
+Starting 9.14 a tool is available to automatically:
+
+* deploy PHP FPM 5.6 configuration in ``/etc/opt/rh/rh-php56/php-fpm.d/tuleap.conf``
+* deploy Nginx configuration in ``/etc/nginx`` (listen on ~*:80 + all front configuration)
+* update apache configuration (listen on 127.0.0.1:8080 + disable SSL)
+
+First, stop all services (``service tuleap stop`` and ``service httpd stop``)
+
+Run ``/usr/share/tuleap/tools/utils/php56/run.php``
```
Comment (yannis-rossetto) [inline languages/en/administration-guide/run-php56.rst]: use bash block here

File: languages/en/administration-guide/run-php56.rst [hunk 2]
```diff
@@ -3,62 +3,79 @@
...
+
+Then:
+* Update ``/etc/nginx/conf.d/tuleap.conf`` to finish the SSL configuration (point toward the right files).
+* Update ``/etc/tuleap/conf/local.inc`` and set (or add): ``$sys_trusted_proxies = '127.0.0.1';``
 
-Configure apache
-~~~~~~~~~~~~~~~~
+Finally you can restart all services:
+
+#. ``service tuleap start``
```
Comment (yannis-rossetto) [inline languages/en/administration-guide/run-php56.rst]: use bash block here

File: languages/en/administration-guide/run-php56.rst [hunk 3]
```diff
@@ -3,62 +3,79 @@
...
+#. ``service httpd start``
+#. ``service rh-php56-php-fpm start``
+#. ``service nginx start``
+
+You should also ensure that fpm and nginx will be started at machine boot:
+
+#. ``chkconfig rh-php56-php-fpm on``
```
Comment (yannis-rossetto) [inline languages/en/administration-guide/run-php56.rst]: use bash block here

File: languages/en/administration-guide/run-php56.rst [hunk 4]
```diff
@@ -111,117 +128,39 @@ Then, in ``/etc/opt/rh/rh-php56/php-fpm.d/www.conf``:
...
 
-You can check that tracker pages (for instance) are served by php 5.6 by checking header's responses.
+* The entry point for all HTTP & HTTPS traffic is nginx (>=1.10).
+  The configuration base dir is ``/etc/nginx`` and the logs are located in ``/var/log/nginx`` and you will find an access log as well as an error_log.
+  PHP errors (like blank pages & all) will be found in nginx error_log.
+  On the very plus size of nginx, configuration can be updated without server shutdown. After a configuration update,
+  issue a ``service nginx configtest`` and if ok ``service nginx reload``
+* Apache is still there to serve subversion traffic (mod_dav_svn only exists for apache) and mailman (cgi). Expect a low
+  traffic there.
+* PHP is served by php-fpm (no longer mod_php). It's a standalone daemon that you can manipulate with ``service rh-php56-php-fpm COMMAND``.
```
Comment (yannis-rossetto) [inline languages/en/administration-guide/run-php56.rst]: use bash block here

File: languages/en/deployment-guide/intro.rst [hunk 5]
```diff
@@ -35,6 +35,18 @@ When the usage of your server is low, you can launch the following script:
 The script will display how much comments it will have to store. You can safely cancel the execution 
 of the script at any moment with ``ctrl-c`` and relaunch it later.
 
+PHP 5.6 / nginx
+---------------
+
+Tuleap now comes with ``php 5.6`` (from Software Collections) and ``nginx`` packages by default. All new installations
+defaults to this setup.
+
+You can switch to php 5.6 by using ``/usr/share/tuleap/tools/utils/php56/run.php``. This will configure nginx as the new
```
Comment (yannis-rossetto) [inline languages/en/deployment-guide/intro.rst]: use bash block here

**Comment by vaceletm** [comment]
Ok, but I won't have time to fix that before the release

**Comment by yannis-rossetto** [comment]
Ok, We can do 2 things:

* I merge this PR and we do the visual changes after
* I rework on this PR to add these visual effects and someone else review it

What do you prefer ?

File: languages/en/administration-guide/run-php56.rst (hunk 1 above)
Comment (nterray) [reply languages/en/administration-guide/run-php56.rst]: done

File: languages/en/administration-guide/run-php56.rst (hunk 2 above)
Comment (nterray) [reply languages/en/administration-guide/run-php56.rst]: done

File: languages/en/administration-guide/run-php56.rst (hunk 3 above)
Comment (nterray) [reply languages/en/administration-guide/run-php56.rst]: done

File: languages/en/administration-guide/run-php56.rst (hunk 4 above)
Comment (nterray) [reply languages/en/administration-guide/run-php56.rst]: done

File: languages/en/deployment-guide/intro.rst (hunk 5 above)
Comment (nterray) [reply languages/en/deployment-guide/intro.rst]: done

**Comment by nterray** [comment]
> Ok, We can do 2 things:
>
>    I merge this PR and we do the visual changes after
>    I rework on this PR to add these visual effects and someone else review it
>
> What do you prefer ?

I contributed to the PR, so that you can continue the review.

**Pull request merged**